### PR TITLE
Admin attach doc restyle

### PR DESCRIPTION
**Pull request by hammerfunk**
Somethings aren't working correctly/not implemented since this restyle

- Firstly the title and the admin only checkbox are placeholder
- the admin only checkbox is for admins only
- the upload button uploads a file but if you did not click the final attach and refresh the file is uploaded anyways
- same with the cancel button, currently it just closes the pop up, it does not remove the uploaded file

### CODE REVIEW

File: app/assets/javascripts/admin/form_answers.js.coffee [hunk 1]
```diff
@@ -22,10 +22,24 @@ ready = ->
...
+      $("#new_form_answer_attachment").addClass("uploaded-file")
+      $("#new_form_answer_attachment ul").append(result)
+
+      # Show/hide the attach document form
+      $("#new_form_answer_attachment").closest(".sidebar-section").addClass("show-attachment-form")
+
+  # Move the attach document button
+  $(".attachment-link").removeClass("if-js-hide")
+  $(".attachment-link").addClass("btn btn-default btn-block btn-attachment")
+  $(".attachment-link").prepend("<span class='glyphicon glyphicon-paperclip'></span> Attach Document")
```
Comment (matthewford) [inline app/assets/javascripts/admin/form_answers.js.coffee]: Line exceeds maximum allowed length

File: spec/features/admin/form_answers/attachments_management_spec.rb [hunk 2]
```diff
@@ -14,10 +14,10 @@
     visit admin_form_answer_path(form_answer)
   end
 
-  it 'adds the attachment' do
-    attach_file 'form_answer_attachment_file', "#{Rails.root}/spec/fixtures/cat.jpg"
-    click_button 'Attach Document'
-    expect(page).to have_selector('.form_answer_attachment', count: 1)
+  it "adds the attachment" do
+    attach_file "form_answer_attachment_file", "#{Rails.root}/spec/fixtures/cat.jpg"
+    click_button "Attach"
```
Comment (matthewford) [inline spec/features/admin/form_answers/attachments_management_spec.rb]: I prefer to scope the specs so they don't break with copy changes, so something like

within ".form" do 
click_button ".submit"
end

File: spec/features/admin/form_answers/attachments_management_spec.rb [hunk 3]
```diff
@@ -14,10 +14,12 @@
     visit admin_form_answer_path(form_answer)
   end
 
-  it 'adds the attachment' do
-    attach_file 'form_answer_attachment_file', "#{Rails.root}/spec/fixtures/cat.jpg"
-    click_button 'Attach Document'
-    expect(page).to have_selector('.form_answer_attachment', count: 1)
+  it "adds the attachment" do
+    within "#new_form_answer_attachment" do
```
Comment (matthewford) [inline spec/features/admin/form_answers/attachments_management_spec.rb]: Trailing whitespace detected.

File: spec/features/admin/form_answers/attachments_management_spec.rb (hunk 2 above)
Comment (hammerfunk) [inline spec/features/admin/form_answers/attachments_management_spec.rb]: Cool I've done something similar now

File: spec/features/admin/form_answers/attachments_management_spec.rb [hunk 4]
```diff
@@ -14,10 +14,12 @@
...
+      attach_file "form_answer_attachment_file", "#{Rails.root}/spec/fixtures/cat.jpg"
+      find(".spec-submit").click
```
Comment (matthewford) [inline spec/features/admin/form_answers/attachments_management_spec.rb]: The name of the class shouldn't be spec specific like this, I would just use it btn-submit

**Pull request merged**